PACKAGE.md

```diff
@@ -87,33 +87,68 @@ It should typically be enough to run `osc ar` to add new and delete removed file
 ```bash
 osc commit
 ```
+## Step 7. Submit Requests to openSUSE Factory and SLES
 
-## Step 7. Submit Requests to OpenSUSE and SLES
+To get a maintenance request accepted, each changelog entry needs to have at
+least one reference to a bug or feature request like `bsc#123` or `fate#123`.
 
-To get a maintenance request accepted, each changelog entry needs to reference a bug or feature
-request with `bsc#123` or `fate#123`.
+Note: If you want to disable automatic changes made by osc (e.g. License string)
+      use the `--no-cleanup` switch.
 
-### OpenSUSE Factory
+### Factory First
 
 To submit a request to openSUSE Factory, issue this commands in the console:
 
 ```bash
-osc sr systemsmanagement:SCC SUSEConnect openSUSE:Factory --no-cleanup
+osc sr systemsmanagement:SCC SUSEConnect openSUSE:Factory
 ```
 
-### SLES (Internal Build Service)
+### Submit packages to the Internal Build Service (SLES)
```

Comment:
Better: 
`### Submit maintenance updates for SLES to the Internal Build Service`